website/blog/2023-08-20-databend-weekly-107.md

```diff
@@ -0,0 +1,148 @@
+
+The following example shows how to create a Hive Catalog with MinIO as the underlying storage service:
+
+```sql
+CREATE CATALOG hive_ctl 
+TYPE = HIVE 
+CONNECTION =(
+    ADDRESS = '127.0.0.1:9083' 
+    URL = 's3://warehouse/' 
+    AWS_KEY_ID = 'admin' 
+    AWS_SECRET_KEY = 'password' 
+    ENDPOINT_URL = 'http://localhost:9000/'
+)
+```
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [Issue #12407 | Feature: Add storage support for Hive catalog](https://github.com/datafuselabs/databend/issues/12407)
+- [PR #12469 | feat: Add storage params in hive catalog](https://github.com/datafuselabs/databend/pull/12469)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Using `gitoxide` to Speed Up Git Dependency Downloads
+
+`gitoxide` is a high-performance, modern Git implementation written in Rust. Utilizing the `gitoxide` feature of cargo (Unstable), the `gitoxide` crate can replace `git2` to perform various Git operations, thereby achieving several times performance improvement when downloading crates-index and git dependencies.
+
+Databend has recently enabled this feature for `cargo {build | clippy | test}`` in CI. You can also try to add the -Zgitoxide option to speed up the build process during local development:
+
+```bash
+cargo -Zgitoxide=fetch,shallow-index,shallow-deps build
+```
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [chore(ci): cargo build with -Zgitoxide](https://github.com/datafuselabs/databend/pull/12504)
+- [The Cargo Book | Unstable - gitoxide](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#gitoxide)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- `VALUES` clause can be used without being combined with `SELECT`.
+- Setting default values is supported when modifying column type.
+- **Virtual Column** support has been added for tables in Parquet format.
+- Automatic reclustering of tables is now supported after performing write operations (`COPY INTO` and `REPLACE INTO`).
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Enhancing `infer_schema` Functionality to Support All File Locations
+
+Currently, it is possible to query files using file locations or stages in our database. Examples of this are:
+
+```sql
+select * from 'fs:///home/...';
+select * from 's3://bucket/...';
+select * from @stage;
+```
+
+However, when it comes to invoking the `infer_schema` function, it is limited to only staged files. For example:
+
+```sql
+select * from infer_schema(location=>'@stage/...');
+```
+
+When trying to use `infer_schema` with other file locations, it causes a panic:
```

Comment:
When attempting to use infer_schema with other file locations, it leads to a panic: